specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2021-10-14/livetoken_API.json

```diff
@@ -36,44 +36,6 @@
     }
   },
   "paths": {
```

Comment:
Removing livetoken_API.json is the right fix.

Reply:
livetoken_API.json has another path which is not present anywhere else (/{resourceUri}/providers/Microsoft.Insights/generatelivetoken) with operationId = LiveToken_Get). I thought the idea was to remove the duplicate path, not all paths?